style.css

```diff
@@ -0,0 +1,231 @@
+/* Global Layout Set-up */
+ {
```

Comment:
This currently, isn't targeting any elements. If you want to target everything, use the wild card *